.github/workflows/R-CMD-check.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - {os: macos-latest,   r: 'release'}
+          - {os: macos-13,   r: 'release'} #TODO change back to macos-latest some day when ChemmineOB works on M1
```

Comment:
Just do both maybe?  (I guess we'll have a red x until things work again though)